qa/L0_infer_zero/infer_zero_test.py

```diff
@@ -41,17 +41,34 @@
 
 TEST_SYSTEM_SHARED_MEMORY = bool(int(os.environ.get("TEST_SYSTEM_SHARED_MEMORY", 0)))
 TEST_CUDA_SHARED_MEMORY = bool(int(os.environ.get("TEST_CUDA_SHARED_MEMORY", 0)))
+BACKENDS = os.environ.get("BACKENDS", "graphdef savedmodel onnx libtorch")
 
 
 class InferZeroTest(tu.TestResultCollector):
     def _full_zero(self, dtype, shapes):
         # 'shapes' is list of shapes, one for each input.
+        for backend in BACKENDS.split(" "):
+            if backend == "graphdef" or backend == "savedmodel":
+                if not tu.validate_for_tf_model(
+                    dtype, dtype, dtype, shapes[0], shapes[0], shapes[0]
+                ):
+                    return
+
+            if backend == "onnx":
+                if not tu.validate_for_tf_model(
```

Comment:
validate_for_onnx_model?

Reply:
good catch, fixed.

Reply:
A further refactor could be something like this, if you think it would be a nice change:
```python
VALIDATION_FNS = {
  "onnx": tu.validate_for_onnx_model,
  # ...
}

# ...

            # object models do not exist right now for PyTorch
            if backend == "libtorch" and dtype == "object":
                    return

            if not VALIDATION_FNS[backend](dtype, dtype, dtype, shapes[0], shapes[0], shapes[0]):
                return
```

Up to you though

Reply:
good suggestion, done.